src/client.cpp

```diff
@@ -533,9 +547,139 @@ PVR_ERROR GetChannelGroupMembers(ADDON_HANDLE handle, const PVR_CHANNEL_GROUP &g
   return VuData->GetChannelGroupMembers(handle, group);
 }
 
+PVR_ERROR GetStreamReadChunkSize(int* chunksize)
+{
+  if (!chunksize)
+    return PVR_ERROR_INVALID_PARAMETERS;
+  *chunksize = m_streamReadChunkSize;
+  return PVR_ERROR_NO_ERROR;
+}
+
+/* live stream functions */
+bool OpenLiveStream(const PVR_CHANNEL &channel)
+{
+  if (!VuData || !VuData->IsConnected())
+    return false;
+
+  if (!VuData->OpenLiveStream(channel))
+    return false;
+
+  //const Settings &settings = DvbData->GetSettings();
```

Comment:
Another leftover from stuff you copied from dvbviewer. Pleaes this line as it is already commented out.